src/wp-includes/script-loader.php

```diff
@@ -2730,6 +2735,45 @@ static function( $a, $b ) {
 	}
 }
 
+/**
+ * Make URLs relative to the WordPress installation.
+ *
+ * @since 5.9.0
+ * @access private
+ *
+ * @param string $css            The CSS to make URLs relative to the WordPress installation.
+ * @param string $stylesheet_url The URL to the stylesheet.
+ *
+ * @return string The CSS with URLs made relative to the WordPress installation.
+ */
+function _wp_normalize_relative_css_links( $css, $stylesheet_url ) {
+	$has_src_results = preg_match_all( '#url\s*\(\s*[\'"]?\s*([^\'"\)]+)#', $css, $src_results );
```

Comment:
Just curious: is there any reason why `preg_replace_callback()` wasn't used here? If it was, then it could eliminate the last `str_replace()` in the `foreach` loop. I've just opened a ticket and patch which shows the performance of this function can be improved >2x: https://core.trac.wordpress.org/ticket/58069

Reply:
PR to make this change: https://github.com/WordPress/wordpress-develop/pull/4297

Reply:
There's no particular reason... I think at the time, it just didn't cross my mind to use `preg_replace_callback()` 👍